### ISSUE TITLE
[bug] `<rh-tabs>` Wonky unexpected tabbing behavior (roving tab index) in catalog app

### ISSUE THREAD

**Issue by markcaron** [RedHat-UX/red-hat-design-system#1578]
On catalog.redhat.com, tabbing into and through `<rh-tabs>` isn't behaving as expected (based on RTI).

### Examples:

1. [Example 1](https://catalog.redhat.com/solutions/detail/4111a0abefb743ea848d61b23719c69a)
2. [Example 2](https://catalog.redhat.com/partners/detail/lenovo)

Expected behavior: [Tabs demo](https://ux.redhat.com/elements/tabs/demo/)

The `<rh-tabs>` version being imported is `1.4.0` from `@rhds/elements/":"https://www.redhatstatic.com/dx/v1-alpha/@rhds/elements@1.4.0/elements/`

### GitLab code links:
- [Imports](https://url.corp.redhat.com/3089793)
- [Solution Detail](https://url.corp.redhat.com/1e780f4)
- [Tabs](https://url.corp.redhat.com/53542c9)

**Comment by adamjohnson** [comment]
Looks like there are [two `<style>` tags](https://gitlab.cee.redhat.com/customer-platform/ecosystem-catalog/-/blob/main/public/index_template.html#L121-194) in between the closing `</head>` and opening `<body>` tags. These should be moved into the `<head>` according to the [W3C HTML Validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fcatalog.redhat.com%2Fsolutions%2Fdetail%2F4111a0abefb743ea848d61b23719c69a).

Not sure if this will fix it but it'll provide us a good idea of what to look at next.

**Comment by bennypowers** [comment]
repro: https://codepen.io/bennyp/pen/PovWOVj?editors=1000
